apps/smart-contracts/core/test/FeeRebateHook.test.ts

```diff
@@ -32,6 +33,10 @@ describe('=> FeeRebateHook', () => {
   })
 
   describe('initial state', () => {
+    it('does not have treasury set', async () => {
+      expect(await feeRebateHook.treasury()).to.equal(ethers.constants.AddressZero)
```

Comment:
```suggestion
      expect(await feeRebateHook.treasury()).to.eq(ZERO_ADDRESS)
```
Not a big deal, but we already have ZERO_ADDRESS, and please be consistent with using `eq` instead of `equal`